.github/workflows/functional-networking.yaml

```diff
@@ -0,0 +1,59 @@
+name: functional-networking
+on:
+  pull_request:
+    paths:
+      - '^.*networking.*$'
+      - '.github/workflows/functional-networking.yaml'
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  functional-networking:
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["master"]
+        openstack_version: ["master"]
+        ubuntu_version: ["20.04"]
+        include:
+          - name: "xena"
+            openstack_version: "stable/xena"
+            ubuntu_version: "20.04"
+          - name: "wallaby"
+            openstack_version: "stable/wallaby"
+            ubuntu_version: "20.04"
+          - name: "victoria"
+            openstack_version: "stable/victoria"
+            ubuntu_version: "20.04"
+          - name: "ussuri"
+            openstack_version: "stable/ussuri"
+            ubuntu_version: "18.04"
+          - name: "train"
+            openstack_version: "stable/train"
+            ubuntu_version: "18.04"
+    runs-on: ubuntu-${{ matrix.ubuntu_version }}
+    name: Deploy OpenStack ${{ matrix.name }} with Neutron and run networking acceptance tests
+    steps:
+      - name: Checkout Gophercloud
+        uses: actions/checkout@v2
+      - name: Deploy devstack
+        uses: EmilienM/devstack-action@v0.6
+        with:
+          branch: ${{ matrix.openstack_version }}
+      - name: Checkout go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15'
```

Comment:
This step is also weird. It installs go 1.17 where I would expect it to install 1.15:

```
Run actions/setup-go@v2
  with:
    go-version: ^1.15
    stable: true
    token: ***
Setup go stable version spec ^1.15
Found in cache @ /opt/hostedtoolcache/go/1.17.5/x64
Added go to the path
Successfully setup go version ^1.15
go version go1.17.5 linux/amd64
```

https://github.com/gophercloud/gophercloud/runs/4781723151?check_suite_focus=true#step:4:1

Reply:
I'll investigate these issues later.